src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSBE/EventParam.cs

```diff
@@ -172,6 +180,9 @@ internal override Event ReadEntry(BinaryReaderEx br)
                     case EventType.RetryPoint:
                         return RetryPoints.EchoAdd(new Event.RetryPoint(br));
 
+                    case EventType.AreaTeam:
+                        return AreaTeams.EchoAdd(new Event.AreaTeam(br));
+
```

Comment:
Happy to see updates here. Not sure if this what tk was bemoaning but it'll help.